File: app/src/main/java/com/example/paetz/yacguide/database/Region.java
package com.example.paetz.yacguide.database;

import android.arch.persistence.room.Entity;
import android.arch.persistence.room.PrimaryKey;

@Entity
public class Region {

    @PrimaryKey
    private int id;

    private String name;
    private String country;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getCountry() {
        return country;
    }

    public void setCountry(String country) {
        this.country = country;
    }
}

File: app/src/main/java/com/example/paetz/yacguide/database/Region.kt
package com.example.paetz.yacguide.database

import android.arch.persistence.room.Entity
import android.arch.persistence.room.PrimaryKey

@Entity
class Region {

    @PrimaryKey
    var id: Int = 0

    var name: String? = null
    var country: String? = null
}
